openj9-11-rhel7.yaml

```diff
@@ -75,11 +75,6 @@ osbs:
           - s390x
           - ppc64le
       compose:
-        # used for requesting ODCS compose of type "tag"
-        packages:
-          - java-11-openj9
-          - java-11-openj9-headless
-          - java-11-openj9-devel
```

Comment:
I'm a bit out of the loop here, but...

Are you sure this is the right move? Isn't the intention to pull in not-yet-released-but-signed RPMs into the container build? Has this been discussed?

Reply:
This doesn't seem right.  Adding the content set for CVP validation does not contradict the building process.  Without `packages` you would have to wait until the RPMs are shipped before even building the container, which doesn't make sense here.

Reply:
@jerboaa @yselkowitz Thanks for your comment on the changes. I am sorry that I missed mentioning the context of deleting the packages in the description. Recently, after adding the new openj9 rmps (_openj9-1-for-rhel-8-s390x-rpms_) in the content set it was found that due to having the rpm packages declared explicitly in osbs configuration, instead of installing the packages from the repository - '**openj9-1-for-rhel-8-s390x-rpms**' they are being installed from '**odcs-503706**'. Removing the packages from osbs configuration resolves this issue and installs from the latest openj9 rpms repository. I wasn't aware of the issue @yselkowitz mentioned that we have to wait until the RPMs are shipped before building the images. Could you please suggest if I should add them back in the PR. Thanks.

Reply:
This part of the commit should be dropped.

Reply:
@yselkowitz Thanks for your feedback. The openj9 rpm packages have been added back. Please let me know any further modification is required.

Reply:
hi @yselkowitz and @jerboaa , the openj9 rpms have already been shipped [1] [2] to the repos @mdrafiur added. That's why we are removing them from the osbs. So we should drop the packages.

[1] https://errata.devel.redhat.com/advisory/59109
[2] https://errata.devel.redhat.com/advisory/65143

Reply:
hi @yselkowitz, the new feature coming to errata is that it will not support containers release using rpms not shipped. So we have to ship the RPMs before building and releasing these containers. That feature also triggered the task of shipping these rpms (only internally) a bit earlier, details see [1].

[1] https://projects.engineering.redhat.com/browse/SPMM-1387

Reply:
@levivic This is not what SPMM-1387 is about. RPMs will ship, right, but the ODCS (i.e. container.yml file) enables is to create an RPM erratum, flip to QE for signing, build the container with `--signing-intent release`, file container erratum, attach build, do QE. Once ready ship the RPM erratum then the container erratum. It enables what @yselkowitz says to ship the container at the same time as the RPM erratum. Without ODCS you'll need to wait for the container update to go all the way through to SHIPPED_LIVE before you can even begin *building* the container.

Reply:
During the chat with @jerboaa, he helped me understand the purpose of keeping this block, keeping rpms and containers to be shipped simultaneously. So @mdrafiur, lets keep this code. Thanks @yselkowitz for your feedback as well.